pkg/plugins/platform/local.go

```diff
@@ -854,7 +854,19 @@ func (l *LocalApp) Down(removeData bool) error {
 				}
 			}
 		}
-		dir := filepath.Dir(l.AppConfig.DataDir)
+		// Check to see if the AppRoot exists.
+		// If it does not exist, manually set the data directory to remove.
+		var dir string
+		if fileutil.FileExists(l.AppConfig.AppRoot) {
+			dir = filepath.Dir(l.AppConfig.DataDir)
+		} else {
+			user, err := user.Current()
```

Comment:
This should all be using [GetDdevGlobalDir()](https://github.com/drud/ddev/blob/22498e2eebd17a275a115fddbe844adc0807bafa/pkg/util/utils.go#L70), don't reinvent the wheel.